features/admin.core.v1/components/header.tsx

```diff
@@ -257,24 +248,36 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (props: HeaderPro
                             <Divider className="get-help-dropdown-divider" />
                         </>
                     ) }
-                    { window["AppUtils"].getConfig().extensions.community && (
+                    { window["AppUtils"].getConfig().extensions.getHelp.communityLinks.discord && (
                         <MenuItem
                             className="get-help-dropdown-item"
                             onClick={ () => {
-                                window.open(window["AppUtils"].getConfig().extensions.community, "_blank", "noopener");
+                                window.open(window["AppUtils"].getConfig().extensions.getHelp.communityLinks.discord
+                                    , "_blank", "noopener");
                             } }
                         >
                             <>
-                                <ListItemIcon>
-                                    <GenericIcon
-                                        className="spaced-right"
-                                        transparent
-                                        fill="white"
-                                        size="x22"
-                                        icon={ CommunityIcon }
-                                    />
+                                <ListItemIcon className="get-help-icon">
+                                    <DiscordIcon />
+                                </ListItemIcon>
+                                <ListItemText primary=  { t("extensions:common.help.communityLinks.discord") } />
+                            </>
+                        </MenuItem>
+                    ) }
+                    { window["AppUtils"].getConfig().extensions.getHelp.communityLinks.stackOverflow && (
+                        <MenuItem
+                            className="get-help-dropdown-item"
+                            onClick={ () => {
+                                window.open(window["AppUtils"].getConfig()
+                                    .extensions.getHelp.communityLinks.stackOverflow
+                                , "_blank", "noopener");
+                            } }
+                        >
+                            <>
+                                <ListItemIcon className="get-help-icon">
+                                    <StackOverflowIcon />
                                 </ListItemIcon>
-                                { I18n.instance.t("extensions:common.help.communityLink") }
+                                <ListItemText primary=  { t("extensions:common.help.communityLinks.stackOverflow") } />
```

Comment:
```suggestion
                                <ListItemText primary={ t("extensions:common.help.communityLinks.stackOverflow") } />
```

Reply:
fixed with https://github.com/wso2/identity-apps/pull/6700/commits/2f9ead44743ee603856ee9d29924c6595497540a

Reply:
This formatting issue seems to be still there.